canonical-websites/snapcraft.io
# Update run script to version 2.5.0

# Summary

Update run script to last version (2.5)
The code is generated by yoeman (I didn't add my-self all this code :) ) This is why I just added the QA label

# QA

- Checkout to this branch
- `./run`
- http://0.0.0.0:8004/
- Make sure the snapcraft.io is accessible
